en/starter/exposeport.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Would this content be better as part of the `leverageDocker.md` page - so that all Docker content is in one place?